example/README.md

```diff
@@ -0,0 +1,83 @@
+## Example
+
+This example app demonstrates how to use the `io-react-native-wallet` library to interact with the IO Wallet.
```

Comment:
IO Wallet is the internal project name. What do you think to use a more generic `Digital Identity Wallet`?

Reply:
The example app does in fact interact with our internal project as it's supposed to log into IO and communicates with io-backend. My idea was to put emphasis on that.